cli/src/main.rs

```diff
@@ -643,6 +674,15 @@ fn list_kdl(context: &Context) {
                 })
                 "\"\n"
             }
+            if let Some(xwayland_primary) = head.xwayland_primary {
+                "  xwayland_primary \""
+                (if xwayland_primary {
+                    "true"
+                } else {
+                    "false"
```

Comment:
These need to be `#true` / `#false`.

Reply:
hmm, it isn't for adaptive_sync_support or adaptive_sync, does this only apply to pure booleans?

Reply:
in any case, should be resolved.